app/assets/javascripts/Components/graders_manager.jsx

```diff
@@ -19,13 +22,34 @@ class GradersManager extends React.Component {
       anonymize_groups: false,
       hide_unassigned_criteria: false,
       sections: {},
+      isGradeerDistributionModalOpen: false,
+      weightings: {},
     };
   }
 
   componentDidMount() {
     this.fetchData();
   }
 
+  showDistributionModal = () => {
```

Comment:
different names should be as consistent as possible; a better name here is `openGraderDistributionModal`